engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-subscriptions.html

```diff
@@ -1,5 +1,25 @@
 <span page-title ng-model="activationKey">{{ 'Subscriptions for Activation Key:' | translate }} {{ activationKey.name }}</span>
 
+<section>
+  <h4 translate>Activation Key Type:</h4>
+
+  <div class="detail">
```

Comment:
I think this class should be 'details'.  It makes the 'editable' interaction not span the entire width.